recipes/ucsc-bedgraphtobigwig/patch1.patch

```diff
@@ -0,0 +1,19 @@
+--- userApps/kent/src/lib/htmshell.c.old	2022-10-28 08:32:08.344445121 +0000
++++ userApps/kent/src/lib/htmshell.c	2022-10-28 08:32:31.708262420 +0000
+@@ -711,11 +711,11 @@
+ puts("Status: 400\r");
+ puts("Content-Type: text/plain; charset=UTF-8\r");
+ puts("\r");
+-if (format != NULL && args != NULL)
+-    {
+-    vfprintf(stdout, format, args);
+-    fprintf(stdout, "\n");
+-    }
++if (format != NULL)
```

Comment:
This has been fixed upstream several months ago - https://github.com/ucscGenomeBrowser/kent/commit/8b044625d9cdd9dfaec89ff32d0ecb069b7c0565
This fix is in tag `[v463_preview](https://github.com/ucscGenomeBrowser/kent/releases/tag/v463_preview)`

Reply:
Changing the sources would be a big change indeed! Let's continue with the patch for now!
Please apply the suggestion - https://github.com/bioconda/bioconda-recipes/pull/46516#discussion_r1529823216